src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/NonEmptyAtclauseDescriptionCheck.java

```diff
@@ -67,44 +67,153 @@
  * <pre>
  * class Test
  * {
- * &#47;**
- * * Violation for param "b" and at tags "deprecated", "throws".
- * * &#64;param a Some javadoc // OK
- * * &#64;param b
- * * &#64;deprecated
- * * &#64;throws Exception
- * *&#47;
- * public int method(String a, int b) throws Exception
+ *  // Violation for param "b" and at tags "deprecated", "throws".
+ *  &#47;**
+ *  * Some summary.
+ *  * &#64;param a Some javadoc // OK
+ *  * &#64;param b
+ *  * &#64;deprecated
+ *  * &#64;throws Exception
+ *  *&#47;
```

Comment:
there is no `return` please add, you have it in others.
java doc comment should be idential by structure in all examples.